.github/workflows/conformance-ipsec-e2e.yaml

```diff
@@ -319,14 +319,22 @@ jobs:
             --junit-property github_job_step="Run tests (${{ join(matrix.*, ', ') }})" \
             --flush-ct
 
-      - name: Rotate IPsec Key & Test (${{ join(matrix.*, ', ') }})
+      - name: Setup conn-disrupt-test before rotating (${{ join(matrix.*, ', ') }})
+        uses: ./.github/actions/conn-disrupt-test-setup
+
+      - name: Rotate IPsec Key (${{ join(matrix.*, ', ') }})
         uses: ./.github/actions/ipsec-key-rotate
```

Comment:
This action is also used in `conformance-eks.yaml`, and should be adapted there as well to avoid losing coverage.